src/dodal/devices/dcm.py

```diff
@@ -36,4 +43,16 @@ def __init__(
             self.perp_temp = epics_signal_r(float, prefix + "TEMP6")
             self.perp_sub_assembly_temp = epics_signal_r(float, prefix + "TEMP7")
 
+            self.crystal_metadata_usage, _ = soft_signal_r_and_setter(
+                str, initial_value=crystal_metadata.usage
+            )
+            self.crystal_metadata_type, _ = soft_signal_r_and_setter(
+                str, initial_value=crystal_metadata.type
+            )
+            self.crystal_metadata_reflection, _ = soft_signal_r_and_setter(
+                Sequence[int],
+                initial_value=list(crystal_metadata.reflection),  # type: ignore
+            )
+            # todo note this fails at i18 for now, waiting for an EPICS fix
+            self.crystal_metadata_d_spacing = epics_signal_r(float, "DSPACING")
```

Comment:
Must: We need to read `DSPACING:RBV` as this contains the read value, `DSPACING` is currently 0

Reply:
I thought that :RBV is added automatically in epics_signal_r but ok, will amend this

Reply:
changed